src/alt-gosling-model/alt-text/color.ts

```diff
@@ -0,0 +1,83 @@
+import { GetColorName } from 'hex-color-to-color-name';
+
+// Converting colors to names
+
+export function getColorName(color: string, simple?: boolean): string {
+    console.log("color", "simple", color, simple);
+    color = color.replace('#', '');
+    if (simple === undefined || simple === false) {
```

Comment:
I think you can simply do the following that will capture both cases:

```suggestion
    if (!simple) {
```